xml/soc-operations-gpu-passthrough.xml

```diff
@@ -20,12 +20,14 @@
    &productnumber; &compnode;: preparing the &compnode;, preparing &o_comp; and
    &o_img;.
   </para>
-  <orderedlist>
-   <listitem xml:id="crow-pci-passthrough-prep-node">
-    <para>
-     <emphasis role="bold">Preparing the &compnode;</emphasis>
-    </para>
-    <procedure>
+  <note>
+    <para>The following procedures need to be followed sequentially to ensure
+      GPU passthrough is setup correctly.</para>
+  </note>
```

Comment:
How about instead of using a note here, you change the intro para above? E.g.:

```
  <para>
   To leverage PCI passthrough on a &cloud;
   &productnumber; &compnode;, follow the below procedures in sequence:
  </para>
  <orderedlist>
   <listitem>
    <para>
     <xref linkend="crow-pci-passthrough-prep-node"/>
    </para>
   </listitem>
   <listitem>
    <para>
     <xref linkend="flavor-creation"/>
    </para>
  </listitem>
 </orderedlist>
```